### ISSUE TITLE
Version 4.7.6 for macOS

### ISSUE THREAD

**Issue by mtebele** [OpenCobolIDE/OpenCobolIDE#416]
The latest version dmg file is missing.

**Comment by GitMensch** [comment]
Correct, seems like @ColinDuquesnoy did not build the dmg file (the last one was 4.7.4 [4.7.5 was a bugfix release for builds from source and therefore did not need a dmg]), at least it isn't in the releases, neither here nor in https://launchpad.net/cobcide
As the 4.7.7 milestone is at 0% I guess he'll add the dmg. Not sure but you may be able to produce it yourself from source?

**Comment by mtebele** [comment]
Better if @ColinDuquesnoy build it from source, I have never done that.

**Comment by GitMensch** [comment]
@ColinDuquesnoy in any case: please point to a short HowTo for building a DMG from source. Thank you.

**Comment by ColinDuquesnoy** [comment]
I am having issues building the dmg on my mac (I recently had an issue during an OSX upgrade and had to reinstall the system, since then the built dmg file just does not work).

For now, I'd suggest running from sources:

1. Install Python3 and PyQt5 using homebrew: `brew install python3 pyqt5`
2. Install OpenCobolIDE using the python package manager: `pip3 install opencobolide`

To start the IDE, open a terminal and type `opencobolide`

The issue I have seem to be related to cx_freeze, when I'll work on v4.7.7 I'll try again by testing different versions of python and cx_freeze in a virtual env.
